index.html

```diff
@@ -54,10 +59,10 @@ <h1 class="banner__heading">WITH <span class="banner__highlight">BINABOX</span>
         <div class="catalog__header">
           <h2 class="catalog__title">Catalog</h2>
           <div class="catalog__controls">
-            <a href="#" class="catalog__view-all">SEE ALL</a>
+            <a href="#" class="catalog__view-all button--secondary">SEE ALL</a>
```

Comment:
Modifier is an **extra class name** which you add to a block/element DOM node. Add modifier classes **only** to blocks/elements **they modify**, and keep the original class.

So, modifier need modified class in the same class attribute